install_and_configure.yml

```diff
@@ -0,0 +1,121 @@
+      user:
+        name: epidose
+        groups: testgroup
+        append: yes
+        state: present
+        createhome: yes
+
+    - name: deploy ssh key
+      tags: production, development
+      authorized_key:
+        key: "{{ lookup('file', ssh_key) }}"
+        user: epidose
+
+    - name: install libraries and packages
+      tags: production, development
+      become: yes
+      become_user: epidose
+      apt:
+        name: ['git', 'libbluetooth-dev', 'libglib2.0-dev', 'python3-dev', 'python3-setuptools', 'shellcheck', 'sqlite3', 'virtualenv', 'dh-virtualenv', 'debhelper', 'supervisor']
+
+    - name: setup production environment
+      tags: production
+      become: yes
+      become_user: epidose
+      shell: |
+       cd /home/epidose
+       git clone https://github.com/dspinellis/epidose
+       cd epidose      
+       virtualenv venv -p /usr/bin/python3
+       . venv/bin/activate
+       pip3 install -e ".[dev,test,deploy]"
+       make package
+       sudo make install
+      register: setup_env_output
+
+    - name: setup development environment
+      tags: development
+      become: yes
+      become_user: epidose
+      shell: |
+       cd /home/epidose
+       git clone https://github.com/dspinellis/epidose
+       cd epidose    
+       virtualenv venv -p /usr/bin/python3
+       . venv/bin/activate
+       pip3 install -e ".[dev,test,deploy]"
+       pre-commit install
+       git remote add dp3t https://github.com/DP-3T/reference_implementation.git
+       git fetch
+       make package
+       sudo make install
+      register: setup_env_output
+
+    - debug:
+         msg: "{{ setup_env_output.stdout_lines|list }}"
+      tags: production, development
+
+    - name: installl fake-hwclock to /sbin
+      tags: production, development
+      become: yes
+      become_user: root
+      shell: cp /home/epidose/epidose/epidose/device/setup/fake-hwclock /sbin
+
+    # The configurations below are necessary to allow wireless roaming
+    - name: enable network roaming
+      tags: production, development
+      become: yes
+      become_user: root
+      shell: |
+       cat > /etc/network/interfaces.d/wlan0 << EOF
+       # The wireless network interface
+       allow-hotplug wlan0
+       iface wlan0 inet manual
+       wpa-roam /etc/wpa_supplicant/wpa_supplicant.conf
+       EOF
+
+    # Setup Eduroam network configurations
+    - name: setup Eduroam
+      tags: production, development
+      become: yes
+      become_user: root
+      blockinfile: |
+        dest=/etc/wpa_supplicant/wpa_supplicant.conf backup=yes
+        content='network={
+            ssid="epidose"
+            psk="{{ epidose_network_psk }}" }
+            
+            network={
+                    ssid="eduroam"
+                    scan_ssid=1
+                    key_mgmt=WPA-EAP
+                    eap=PEAP
+                    identity="epidose"
+                    password="{{ eduroam_network_psk }}"
```

Comment:
No visible password, nice!

Reply:
Yes, I will write the appropriate variables and configurations in the README.md file.

Reply:
Great!  Please include those changes in this PR.